platform/features/layout/src/FixedController.js

```diff
@@ -21,8 +21,22 @@
  *****************************************************************************/
 
 define(
-    ['./FixedProxy', './elements/ElementProxies', './FixedDragHandle'],
-    function (FixedProxy, ElementProxies, FixedDragHandle) {
+    [
+        'lodash',
+        './FixedProxy',
+        './elements/ElementProxies',
+        './FixedDragHandle',
+        '../../table/src/TelemetryCollection',
+        '../../../../src/api/objects/object-utils'
```

Comment:
Mostly for real-time telemetry. It needs to buffer incoming telemetry which may fall within bounds on a subsequent tick. Sometimes telemetry timestamps can be out of sync with the tick source that is advancing the time conductor. `TelemetryCollection` already neatly handles the nuances here.